buildstockbatch/schemas/v0.2.yaml

```diff
@@ -35,9 +35,9 @@ aws-job-environment:
   memory: int(min=1024, required=False)
 
 aws-emr-spec:
-  master_instance_type: str(required=False)
-  slave_instance_type: str(required=False)
-  slave_instance_count: int(min=1, required=False)
+  manager_instance_type: str(required=False)
+  worker_instance_type: str(required=False)
+  worker_instance_count: int(min=1, required=False)
```

Comment:
Same here. Since we're making breaking changes to the input format, we should make a v0.3 schema and upgrade to that. I actually have a v0.3 schema going on in #187. Maybe we should wait for that to merge (soon) and make these changes there.

Reply:
Addressed in https://github.com/NREL/buildstockbatch/pull/205/commits/517049db72bbd2f5ab210d85db08a739e421dd86. That sounds good, we can hold off for the next schema to get merged.